src/pages/mentors.jsx

```diff
@@ -1,17 +1,110 @@
 import React from "react";
+import { graphql } from "gatsby";
+import { Col, Row } from "reactstrap";
+import moment from "moment-timezone";
 import Layout from "../layout";
-import Mentor from "../components/Mentor/Mentor";
-import config from "../../data/SiteConfig";
 import SEO from "../components/SEO/SEO";
+import MentorCards from "../components/Cards/MentorCards";
 
-const mentors = () => (
-  <Layout title="Mentors">
-    <Mentor />
-  </Layout>
-);
+function Mentors({ data }) {
+  const projectEdges = data.allTrelloCard.edges;
+  const defaultAvatar = data.file;
 
-export default mentors;
+  console.log(JSON.stringify(projectEdges, null, 2));
+  const mentorList = projectEdges.map(({ node }) => {
+    const {
+      name,
+      labels,
+      custom_fields: customFields,
+      childMarkdownRemark,
+      childCardMedia,
+    } = node;
+    const { html } = childMarkdownRemark;
+    const avatar = childCardMedia ? childCardMedia.localFile : defaultAvatar;
 
-export function Head() {
-  return <SEO title={`Mentors | ${config.siteTitle}`} />;
+    const tags = labels.map((l) => l.name);
+    const githubProfile = customFields
+      .filter((field) => field.idCustomField === "5eb71b3551de3a59ce8d9bd8")
+      .map((field) => field.value.text);
+    const timeZone = customFields
+      .filter((field) => field.idCustomField === "5eb71b53f52d88487f550e83")
+      .map((field) => field.value.text);
+    const countryCode = customFields
+      .filter((field) => field.idCustomField === "5eb71b7081a67c3b58ea67ed")
+      .map((field) => field.value.text.toLowerCase());
+
+    return { name, avatar, tags, html, githubProfile, timeZone, countryCode };
+  });
+
+  return (
+    <Layout title="Mentors">
+      <Col lg="12" className="card-spacing">
+        <Row className="justify-content-center">
+          {mentorList.map((mentor) => (
+            <MentorCards
+              name={mentor.name}
+              avatar={mentor.avatar}
+              tags={mentor.tags}
+              html={mentor.html}
+              githubProfile={mentor.githubProfile}
+              timeZone={moment
+                .tz(moment(), `${mentor.timeZone}`)
+                .format("HH:mm [(GMT] Z[)]")}
```

Comment:
I'm wondering whether this updates on the live website, or whether we bake in the current time as of building the website :thinking:

Reply:
Whenever the page is newly rendered (e.g. on refresh via `F5`), the time is updated correctly.